app-creator/client/src/redux/types/attributes.ts

```diff
@@ -149,10 +156,14 @@ export const sharedAttributes: AttributeMetaData = {
 export const DEFAULT_SHARED_ATTRIBUTES: DefaultAttributesType = Object.keys(
   sharedAttributes
 ).reduce((attributes: DefaultAttributesType, key) => {
-  if (sharedAttributes[key].type === InputType.number) {
+  if (
+    sharedAttributes[key].type === InputType.number &&
+    key !== 'backgroundOpacity'
```

Comment:
Instead of checking background opacity, can we have a list of keys that should end with the `px` unit, and explicitly check against that?